### ISSUE TITLE
Keyboard Navigation in Multi-line Edit Field for Comment Box Not Accessible to Screen Readers Outside of Assignment Submission.

### ISSUE THREAD

**Issue by njsch** [doubtfire-lms/doubtfire-web#314]
This should not be very difficult to fix.  Comments can be added easily when submitting an assignment but when submitting a general comment i.e. when conversing with a tutor, it seems a cross-browser issue that one categorically cannot navigate through the multiline  editable text field by character, word, line, sentence, or paragraph when using an assistive technology solution such as a screen reader.  The only way around this is to copy and paste the text in from elsewhere.  Consequently, my current SIT102 portfolio (which at least still gave me an HD) has all these typos everywhere in the comments and some of the markdown didn't render properly.

Maybe the type of text field used needs to be re-examined.

**Comment by njsch** [comment]
Closing this issue for now as I tried with a new unit and the <div> appears to be working now.  No idea why it didn't work last year, unless @jakerenzella has fixed it and didn't update the issue.  Though it may also be that screen-reader APIs have been updated as JAWS in particular seems to work better with Chrome these days.

**Comment by jakerenzella** [comment]
Hey @njsch I believe I took a look at the code which creates the input fields and updated/changed some packages. Regardless, I'm very glad to hear this is resolved
